Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
+resolver = "2"
```

Comment:
what dis do?

Reply:
https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions

We have edition=“2021” so 2 should be the default but I was having issues and setting resolver = 2 fixed them so..